packages/terra-menu/src/_MenuContent.jsx

```diff
@@ -342,6 +343,23 @@ class MenuContent extends React.Component {
     );
   }
 
+  ariaDescribedByHandle(value, index) {
+    let ariaDescribedByValue;
+    if (MenuUtils.isMac()) {
+      ariaDescribedByValue = value.props.ariaDescribedBy;
+    }
+    if (!MenuUtils.isMac()) {
+      if (!this.props.index && this.props.showHeader && index === 0 && value.props.ariaDescribedBy) {
+        ariaDescribedByValue = `${this.menuTopHeaderId} ${value.props.ariaDescribedBy}`;
```

Comment:
IDs should be comma separated values 
```suggestion
        ariaDescribedByValue = `${this.menuTopHeaderId}, ${value.props.ariaDescribedBy}`;
```

Reply:
JAWS SR skipping menuHeaderId information If we add comma separated between ID's, SR been reading only second ID information.